package.json

```diff
@@ -146,17 +146,21 @@
         "@formatjs/intl-unified-numberformat": "^3.2.0",
         "@hapi/address": "2.1.1",
         "@sambego/storybook-state": "^2.0.1",
-        "@storybook/addon-actions": "^5.3.9",
-        "@storybook/addon-docs": "^5.3.9",
-        "@storybook/addon-info": "^5.3.9",
-        "@storybook/addon-knobs": "^5.3.9",
-        "@storybook/addon-links": "^5.3.9",
-        "@storybook/addon-notes": "^5.3.9",
-        "@storybook/addon-storyshots": "^5.3.9",
-        "@storybook/addon-viewport": "^5.3.9",
-        "@storybook/addons": "^5.3.9",
-        "@storybook/preset-typescript": "^1.1.0",
-        "@storybook/react": "^5.3.9",
+        "@storybook/addon-actions": "^7.5.3",
+        "@storybook/addon-docs": "^7.5.3",
+        "@storybook/addon-essentials": "^7.5.3",
+        "@storybook/addon-interactions": "^7.5.3",
+        "@storybook/addon-knobs": "^7.0.2",
+        "@storybook/addon-links": "^7.5.3",
+        "@storybook/addon-storyshots": "^7.5.3",
+        "@storybook/addon-styling": "^1.3.7",
+        "@storybook/addon-styling-webpack": "^0.0.5",
+        "@storybook/addon-viewport": "^7.5.3",
+        "@storybook/addons": "^7.5.3",
```

Comment:
we had these "registered" in the previous `.storybook/addons.js`: https://github.com/box/box-ui-elements/blob/1b6f4c2885f24caa8945383a3031f628c2371486/.storybook/addons.js

is there anything we need to do for that now that the file is deleted? i.e.
- are the packages still used?
- is everything handled by `main.ts`?

Reply:
i pruned the list and yea